app/presenters/corporate_information_page_presenter.rb

```diff
@@ -1,2 +1,49 @@
 class CorporateInformationPagePresenter < ContentItemPresenter
+  def initialize(content_item, view_context)
```

Comment:
We can probably get rid of the view context by including the link_helper, a la https://github.com/alphagov/frontend/pull/4687/commits/d94fa1399967c20c524e18fbdb8ae1352ff5c095

Reply:
(which in turn means we don't need this initializer at all)

Reply:
I've updated this now in commit [Add corporate information group methods to CorporateInformationPage presenter](https://github.com/alphagov/frontend/pull/4608/commits/a8448d31cbed05af936a21b2c323be72be4bf802)